src/main/java/org/radarcns/managementportal/Source.java

```diff
@@ -32,12 +32,12 @@
 
     @JsonProperty("id")
     private Integer id;
-    @JsonProperty("deviceTypeId")
-    private String deviceTypeId;
-    @JsonProperty("deviceTypeProducer")
-    private String deviceTypeProducer;
-    @JsonProperty("deviceTypeModel")
-    private String deviceTypeModel;
+    @JsonProperty("sourceTypeId")
+    private String sourceTypeId;
+    @JsonProperty("sourceTypeProducer")
+    private String sourceTypeProducer;
+    @JsonProperty("sourceTypeModel")
+    private String sourceTypeModel;
```

Comment:
The MP API also returns a `sourceTypeCatalogVersion`, perhaps better to include it here as well for completeness since the `Sources` API needs the combination of producer, model and version to identify a source.